torch/csrc/jit/passes/onnx/fixup_onnx_conditionals.cpp

```diff
@@ -0,0 +1,42 @@
+#include <torch/csrc/jit/passes/onnx/fixup_onnx_conditionals.h>
+
+namespace torch {
+namespace jit {
+
+namespace onnx{
+using namespace ::c10::onnx;
+}
+
+bool FixupONNXIfs(Block* block) {
+  bool has_nodes = false;
+  for (auto* node : block->nodes()) {
+    has_nodes = true;
+    if (node->kind() == ::c10::onnx::If) {
+      auto* if_node = node;
+      auto* graph = if_node->owningGraph();
+      for (Block* block : node->blocks()) {
+        if (!FixupONNXIfs(block)) {
```

Comment:
Consider separating the check for empty block into another function, e.g. `bool IsEmptyBlock(Block* block)`.
It is a little confusing that `FixupONNXIfs` returns a bool indicating if the block is empty ...